base/applications/rapps/winmain.cpp

```diff
@@ -17,6 +17,7 @@ LPCWSTR szWindowClass = MAINWINDOWCLASSNAME;
 LONG g_Busy = 0;
 
 HWND hMainWnd;
+HWND g_hLastFocus = NULL;
```

Comment:
Maybe this one too could be subject to become a `static CMainWindow::` member :thinking: (like the `m_PendingInstalledViewRefresh` flag)